app/index.js

```diff
@@ -482,7 +482,7 @@ app.on('ready', () => {
         title: 'Save password?',
         message: message,
         buttons: ['Yes', 'No'],
-        defaultId: 0,
+        defaultId: 1,
```

Comment:
If the user wants to use a different password manager or none at all, they can turn it off in `about:preferences`. If they have the password manager turned on, they probably want to save passwords most of the time. So the default should be 'yes'. 

BTW, it's usually safer to use a password manager which saves passwords locally encrypted to disk (which we do) than none at all.